scripts/check_masks.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+import nibabel as nib
+import numpy as np
+import typer
+
+
+def check_masks(directory: Path, file_glob: str = "*.nii.gz"):
+    for f in directory.glob(file_glob):
+        img = nib.load(f"{f}")
+        data = img.get_fdata()
+        max_value = np.max(data)
+        if max_value == 0:
+            print(f"ERROR: {f} mask is empty")
+            return
+
+        min_value = np.min(data[data != 0])
+        if min_value < 1 or max_value != 1:
+            mask = np.zeros_like(data, dtype=np.uint8)
+            mask[data > 0.5] = 1
+            nib.save(nib.Nifti1Image(mask, img.affine), f"{f}")
+            print(f"WARNING: {f} foreground values in range [{min_value}, {max_value}]")
```

Comment:
I would not use print but logging
```python
import logging

logger = logging.getLogger(__file__)

#...
logger.error("%s mask is empty", f)

logger.warning("%s foreground values in range [%s,%s]", f, f"{min_value=}", f"{max_value=}")


```

Reply:
Notice the "=" in the f-strings!

Reply:
is it preferable to do `logger.error("%s mask is empty", f)` versus `logger.error(f"{f} mask is empty")`?